src/p11_atfork.c

```diff
@@ -21,6 +21,7 @@
  */
 
 #include "libp11-int.h"
+#include <unistd.h>
```

Comment:
should I add `#ifndef _WIN32` around?
https://ci.appveyor.com/project/LudovicRousseau/libp11/builds/23751293/job/2o1aangcodd28lg3

not sure what the WIN32 equivalent header file would be...